ngwmn/utils.py

```diff
@@ -0,0 +1,46 @@
+"""
+Utility functions
+
+"""
+from urllib.parse import urljoin
+
+from lxml import etree
+import requests as r
+
+
+def parse_xml(xml_string):
```

Comment:
Wd don't have to do anything now, but I think when we have a critical mass of "utils" to divide them up. Maybe xml_utils, fetch_utils, etc. But no need to do anything now.